frontend/sass/evictionfree/_fonts.scss

```diff
@@ -0,0 +1,51 @@
+@font-face {
+  font-family: "Montserrat Black";
+  font-weight: 900;
+  font-style: normal;
+  src: url("#{$evictionfree-root}/font/montserrat-black-webfont.woff2")
+      format("woff2"),
+    url("#{$evictionfree-root}/font/montserrat-black-webfont.woff")
+      format("woff");
+}
+
```

Comment:
@toolness does it make sense to be copying over these font specifications for the time being? If so, am I setting this up right to work in production?

Reply:
Hmm, unless you feel strongly that we're going to diverge from norent's fonts, I think we should just keep these fonts as-is for now... see my rationale in #1804 for more deets.  But if you're pretty sure we'll be changing things / divering from norent then you can do this.

Reply:
wait, is this PR also copying these files to this new `evictionfree-root` folder?  I would think it needs to, or else the browser won't find these files.  But IMO it will be easier to just reuse the norent css here anyways, unless we plan on diverging.